src/ray/object_manager/test/object_manager_test.cc

```diff
@@ -342,27 +345,27 @@ class TestObjectManager : public TestObjectManagerBase {
     case 0: {
       // Ensure timeout_ms = 0 is handled correctly.
       // Out of 5 objects, we expect 3 ready objects and 2 remaining objects.
-      TestWait(600, 5, 3, /*timeout_ms=*/0, false, false);
+      TestWait(100, 5, 3, /*timeout_ms=*/0, false, false);
```

Comment:
Is decreasing the object size intentional here?

Reply:
Yes, to put it back under the inline objects limit.

Reply:
I see, shouldn't we test both?